src/workerd/api/streams/writable.h

```diff
@@ -158,6 +158,12 @@ class WritableStream: public jsg::Object {
     });
   }
 
+  void serialize(jsg::Lock& js, jsg::Serializer& serializer);
+  static jsg::Ref<WritableStream> deserialize(
+      jsg::Lock& js, rpc::SerializationTag tag, jsg::Deserializer& deserializer);
+
+  JSG_SERIALIZABLE(rpc::SerializationTag::WRITABLE_STREAM);
```

Comment:
Non-blocking nit... can the `JSG_SERIALIZABLE` macro handle adding the `serialize` and `deserialize` boilerplate methods too... just something like `JSG_SERIALIZABLE(WritableStream, rpc::SerializationTag::WRITABLE_STREAM)`

Reply:
Eh, I think it would be confusing if the method declarations are hidden in a macro, given that the definitions are not hidden.

Also, an upcoming change will make it so that these methods can have additional parameters (to request TypeHandlers).